docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -67,93 +67,93 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/semantic_search/website_ingest",
+    "comment": "We use text embeddings in ApertureDB vector store for our documentation chatbot."
   },
   "embeddings_image": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Advanced/Image%20Similarity%20Search",
+    "comment": "Supports multimodal embeddings in general including for videos."
   },
   "embeddings_structured": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/query_language/Reference/descriptor_commands/desc_commands/AddDescriptor",
+    "comment": "ApertureDB vector store can store embeddings from any source but we don't generate them ourselves."
```

Comment:
then this is no support - all the embedding_* fields are about you generating the embeddings for the user data

Reply:
We don't have an example for this yet. So I removed it.